.envs/.local/.django

```diff
@@ -39,3 +39,8 @@ XLI_USER=''
 XLI_PASSWORD=''
 LRM_QA_USER=''
 LRM_QA_PASSWORD=''
+
+#Server Tokens
+#--------------------------------------------------------------------------------
+LRMDEV_TOKEN=''
+LIS_TOKEN=''
```

Comment:
Please follow convention. You can see that we have been using `XLI` not `LIS` and that `LRM` should be followed by an underscore, as in `LRM_DEV_TOKEN` not `LRMDEV_TOKEN`

Reply:
In addition to these, server credentials are also standardized to follow a common convention